Public/Src/Utilities/Configuration/ISandboxConfiguration.cs

```diff
@@ -256,6 +256,6 @@ public interface ISandboxConfiguration
         /// Moreover, this global configuration from cammand line will bypass cache,
         /// which means pips and graph will be cached ignoring paths specified in this configure
         /// </remarks>
-        IReadOnlyList<AbsolutePath> GlobalUnsafeUntrackedScopes { get; }
+        List<AbsolutePath> GlobalUnsafeUntrackedScopes { get; set; }

```

Comment:
The configuration interfaces should remain immutable..

The implementation is mutable...